README.md

```diff
@@ -13,6 +13,10 @@ The purpose of this repository is to help users start creating automated tests u
     ```shell
     git clone --recursive https://github.com/vividus-framework/vividus-starter.git
     ```
+1. Set the BUILD_SYSTEM_ROOT environment variable
```

Comment:
it's not required, unless you already have `BUILD_SYSTEM_ROOT` variable pointing to some other build system

if no `BUILD_SYSTEM_ROOT` variable is set, the starter will try to find an embedded build system and use it

Reply:
@valfirst it's problematic if you have another tool that uses the same variable. maybe creating a unique variable just for vividus will reduce the amount of time debugging

Reply:
@perzeababa makes sense, I've filed a ticket: https://github.com/vividus-framework/vividus/issues/1155

thank you for PR!